packages/components/src/interfaces/internal/Vica.ts

```diff
@@ -0,0 +1,51 @@
+import type { ScriptComponentType } from "~/types"
+
+// NOTE: not all props will be used even if we passed them in
+// as we will override some of them with Isomer's configuration e.g. font-family
+// Nevertheless, keeping them here for reference
+interface VicaWidgetProps {
```

Comment:
abit confused - how will we override? i'm assuming the compiled css will have `font-family` but how will vica pick it up

Reply:
can override directly in https://github.com/opengovsg/isomer/blob/25eee45d5b4748f7b2138bb3acd02f4f5c1c7caa/packages/components/src/templates/next/components/internal/Vica/Vica.tsx#L30-L36

once VICA got back to me, we also ideally want to use the site's `color` to prefill this. For now the only props we know we will override what's passed in is font family